.github/pull_request_template.md

```diff
@@ -1,13 +1,40 @@
-
 <!-- In order to keep off topic discussion to a minimum, it helps if the "work to be done" is already agreed on. -->
```

Comment:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/creating-a-pull-request-template-for-your-repository

Path should be `.github/PULL_REQUEST_TEMPLATE/[filename].md`  (pick a file name).

Reply:
Done.